packages/csharp/package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "cspell-dict-csharp",
   "version": "1.0.1",
-  "description": "C# dictionary for cspell. -- Private until verified",
-  "private": true,
+  "description": "C# dictionary for cspell.",
   "bin": {
     "cspell-dict-csharp-link": "./link.js",
     "cspell-dict-csharp-unlink": "./unlink.js"
   },
   "scripts": {
-    "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 cspell-tools compile \"csharp.txt\" -o .",
+    "build": "cspell-tools compile \"csharp.txt\" -o .",
```

Comment:
Can probably remove cross-env as a devDependency then. I had to add it since it was part of the generator template